src/cookbook.js

```diff
@@ -9,10 +9,16 @@ class Cookbook {
       return recipe.ingredients.find(ingredient => {
         return (ingredient.name.includes(searchText)) ||
         (recipe.name.includes(searchText))
-      });
+      });x
     })
   }
 
+  filterRecipesByTag(tag) {
+    return this.recipes.filter(recipe => {
+      return recipe.tags.includes(tag);
```

Comment:
This creates functionality for the tag filter when *not* favorited

Reply:
thanks also for this addition, I was wondering where to put this method and I think it makes sense to have it here since the cookbook could organize itself by tag